Sources/SQLiteNIO/SQLiteDatabase.swift

```diff
@@ -103,7 +103,7 @@ extension SQLiteDatabase {
     }
 
     /// Async version of ``withConnection(_:)-48y34``.
-    public func withConnection<T>(
+    public func withConnection<T: Sendable>(
```

Comment:
Isn't this breaking?

Reply:
No not in the way we care about. It will produce a warning unless you're in Swift 6 language mode and for now we're not considering that breaking (same with NIO apparently)